cmd/atlas/internal/cmdapi/cmdapi.go

```diff
@@ -23,6 +23,32 @@ var (
 		Use:          "atlas",
 		Short:        "A database toolkit.",
 		SilenceUsage: true,
+		PostRunE: func(cmd *cobra.Command, _ []string) error {
+			// Users may skip update checking behavior.
+			if v := os.Getenv(AtlasNoUpdateNotifier); v != "" {
+				return nil
+			}
+			// Skip if the current binary version isn't set (dev mode).
+			if !semver.IsValid(version) {
+				return nil
+			}
+			path, err := homedir.Expand("~/.atlas/release.json")
```

Comment:
We should consider in the near future passing this configuration logic to `https://github.com/spf13/viper`  (reading files, comparing to env params, and setting "modes" for CLI)

Reply:
its popular but very "big"